native/more/relationship-list-add-button.react.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import type { AppState } from '../redux/redux-setup';
+
+import * as React from 'react';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+import { connect } from 'lib/utils/redux-utils';
+
+import Button from '../components/button.react';
+import { styleSelector, colorsSelector, type Colors } from '../themes/colors';
+
+type ListActionProps = {|
+  onPress: () => void,
+  // Redux state
+  colors: Colors,
+  styles: typeof styles,
```

Comment:
There is no reason to use `stylesSelector`, as none of the styles use any of the custom color functionality